test/apps/iodemo/ucx_wrapper.h

```diff
@@ -138,6 +137,10 @@ class UcxContext {
 
     void recv_io_message();
 
+    bool add_connection(UcxConnection *conn);
+
+    bool remove_connection(UcxConnection *conn);
```

Comment:
return value is not handled anywhere now, can be `void` with new virtual func or maybe assert for debug

Reply:
fixed